oid4vc/oid4vc/routes.py

```diff
@@ -426,9 +429,12 @@ async def supported_credential_create(request: web.Request):
     registered_processors = context.inject(CredProcessors)
     if record.format not in registered_processors.processors:
         raise web.HTTPBadRequest(
-            reason=f"Format {format} is not supported by currently registered processors"
+            reason=f"Format {record.format} is not supported by currently registered processors"
         )
 
+    processor = registered_processors.for_format(record.format)
```

Comment:
We should account for an unknown format. Is that covered in schema validation?

Reply:
Assuming I understand what's happening in the code (and your question lol), this should make sure that `record.format` is present: https://github.com/hyperledger/aries-acapy-plugins/pull/1010/files#diff-b56a50c93b5428feaa477457766d4cd26da0c6025185de2b82e3d339741d985dR345

And this should catch unsupported formats:
https://github.com/hyperledger/aries-acapy-plugins/pull/1010/files#diff-b56a50c93b5428feaa477457766d4cd26da0c6025185de2b82e3d339741d985dR430